apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -0,0 +1,41 @@
+=== Full Site Editing ===
+Contributors: automattic, get_dave, marekhrabe, mppfeiffer, obenland
+Tags: block, blocks, editor, gutenberg, page
+Requires at least: 5.0
+Tested up to: 5.2
+Stable tag: 1.0
```

Comment:
Can we keep this under 1.0? We're not close to this being a release that is production worthy. Ideally this is a 0.X release.